README.md

```diff
@@ -92,6 +92,7 @@ Note: based on these criteria, we would not accept e.g. Whalebrew (but have no p
 ## Tests
 
 Tests can be run with `bundle install && bundle exec rspec`.
+Syntax linting can be run with `brew test-bot --only-tap-syntax --tap homebrew/bundle`.
```

Comment:
```suggestion
Syntax linting can be run with `brew style --tap homebrew/bundle`.
```

is probably a better option here?

Reply:
Ah, didn't know about that one. Fount the `test-bot` command from the ci log. Pushed w/ corrected syntax (your version, but w/o `--tap`).